lib/helper/FileSystem.js

```diff
@@ -60,6 +60,26 @@ class FileSystem extends Helper {
     assert.ok(fileExists(this.file), `File ${name} not found in ${this.dir}`);
   }
 
+  /**
+   * Waits for file to be present in current directory.
+   *
+   * ```js
+   * I.handleDownloads();
+   * I.click('Download large File');
+   * I.amInPath('output/downloads');
+   * I.waitForFile('largeFilesName.txt');
```

Comment:
```suggestion
   * I.waitForFile('largeFilesName.txt', 10); // wait 10 seconds for file
```
An example with a second parameter would be more useful